dist/index.html

```diff
@@ -0,0 +1,62 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <!-- Bootstrap CSS -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-+0n0xVW2eSR5OomGNYDnhzAbDsOXxcvSN1TPprVMTNDbiYZCxYbOOl7+AMvyTG2x" crossorigin="anonymous">
+
+    <!-- favicon -->
+    <link rel="icon" type="image/png" href="https://i.pinimg.com/564x/77/0b/80/770b805d5c99c7931366c2e84e88f251.jpg">
+
+    <title>Weather App</title>
+  </head>
+  <body>
+    <nav class="navbar navbar-expand-lg navbar-dark bg-dark d-flex justify-content-between">
+      <a class="navbar-brand" href="#">Weather App</a>
+    
+        <form id="w-form" class="form-inline my-2 my-lg-0 d-flex justify-content-between">
+          <input id="city" class="form-control mr-sm-2" type="search" placeholder="Search for city" aria-label="Search">
+          <button class="btn btn-outline-success my-2 my-sm-0" type="submit" id="search">Search City</button>
+        </form>
+      </div>
+    </nav>
+
+    <div id="error"></div>
+
+    <div class="content" class="row">
+      <div class="col-md-6 mx-auto text-center bg-dark text-white mt-5 p-5 rounded">
+        <h1 id="w-location"></h1>
+        <h3 id="w-description"></h3>
+        <label class="switch text-center">
+          °C / °F
+          <input id='temp-btn' type="checkbox"  data-toggle="toggle>
```

Comment:
You are using `checkbox` checkout the screenshot from your web page below;
![image](https://user-images.githubusercontent.com/31897434/121206846-b3996900-c870-11eb-8447-712b70a219a3.png)


You should use a toggle control to determine if the result will be shown in Celsius or Fahrenheit ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/search_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/weather-app))